_posts/en/newsletters/2024-01-17-newsletter.md

```diff
@@ -262,8 +262,8 @@ Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo], and
   any onchain fees directly <!-- endogenously --> included in the
   commitment transaction and the presigned HTLC-Success and HTLC-Timeout
   transactions (HTLC-X transactions).  If the funding party doesn't have
-  much funds in the channel and if feerates rise, the funding party may
-  not be able accept a new incoming payment because they don't have
+  many funds in the channel and if feerates rise, the funding party may
```

Comment:
"Much" seems right to me. Isn’t _funds_ an uncountable noun? Either way, perhaps:

```suggestion
  transactions (HTLC-X transactions).  If the funding party is low on
  funds in the channel and if feerates rise, the funding party may
```